app/routes/surveys/components/SurveyEditor/SurveyEditor.js

```diff
@@ -89,6 +89,39 @@ class SurveyEditor extends Component<Props, State> {
     templateTypeSelected: ''
   };
 
+  updateRelativeIndexes = (oldIndex, newIndex, fields) => {
+    fields.move(oldIndex, newIndex);
+  };
+
+  renderQuestions = ({ fields, meta: { touched, error } }) => {
+    return [
```

Comment:
Is there a reason why this return an array?

Reply:
This `renderQuestions` is a prop to redux-form's FieldArray, and I think that just expects an array.